adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -127,25 +127,49 @@ - (void)addDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
     }
   } else if ([delegate respondsToSelector:@selector(isBannerAd)] && [delegate isBannerAd]) {
     @synchronized(_bannerDelegates) {
-      _bannerPlacementID = [delegate.desiredPlacement copy];
-      if (delegate && ![_bannerDelegates objectForKey:delegate.desiredPlacement]) {
-        GADMAdapterVungleMapTableSetObjectForKey(_bannerDelegates, delegate.desiredPlacement,
-                                                 delegate);
+      if (delegate && ![_bannerDelegates objectForKey:delegate.bannerRequest]) {
+        NSEnumerator *enumerator = _bannerDelegates.keyEnumerator;
+        GADMAdapterVungleBannerRequest *bannerRequest = nil;
+        while (bannerRequest = [enumerator nextObject]) {
+          if ([bannerRequest.placementID isEqualToString:delegate.bannerRequest.placementID]) {
+            return NO;
```

Comment:
In the scenario of multiple Banners with same placementID, if the _bannerDelegates already contains a same placementID but the `uniquePubRequestID` is different. This indicates that this is a new Banner request with same placementID, not a refresh Banner request.

Reply:
Got it.